xbmc/windows/GUIWindowLoginScreen.cpp

```diff
@@ -246,12 +246,12 @@ bool CGUIWindowLoginScreen::OnPopupMenu(int iItem)
     return true;
   }
   
-  if (!g_passwordManager.IsMasterLockUnlocked(true))
```

Comment:
Does this explain the issue better? Else maybe look at this [video](https://www.youtube.com/watch?v=NaMEow80dpg&spfreload=10) that I made to show the issue.